handlers/accesscontrol/access-control.go

```diff
@@ -73,21 +78,48 @@ type HitRecords struct {
 	mux  sync.Mutex
 }
 
+// ViewerLimitCache comes from Gate API
+type ViewerLimitCache struct {
+	data map[string]*ViewerLimitCacheEntry
+	mux  sync.RWMutex
+}
+
+type ViewerLimitCacheEntry struct {
+	ViewerLimitPerUser int32
+	UserID             string
+}
```

Comment:
- Do we need to cache these separately from the gate API response?
- Shouldn't this have some `LastRefresh` or similar field?